plugins/no_sir.py

```diff
@@ -0,0 +1,15 @@
+from errbot import BotPlugin
```

Comment:
Please, add tests and no_sir.plug. 
See other .plug files in plugins/ directory and tests in tests/ directory.
New modules can't be added without .plug and testing :cry:

Reply:
k thanks